lib/heuristics/periodic_heuristic.rb

```diff
@@ -791,13 +772,13 @@ def try_to_add_new_point(vehicle_id, day)
       return [nil, [], []] if best_index.nil?
 
       best_index[:end] = best_index[:end] - @services_data[best_index[:id]][:group_duration] + @services_data[best_index[:id]][:durations].first if @same_point_day
-      insert_point_in_route(@candidate_routes[vehicle_id][day], best_index)
+      insert_point_in_route(@candidate_routes[vehicle_id][day], best_index, true, 1)
```

Comment:
insert_point_in_route second parameter is here an index. Elsewhere it seems to be a cost. In the function definition it is named `point_to_add`, i assume it is a `point`. What it is supposed to be ?

Reply:
Here is function definition : 

`insert_point_in_route(route_data, point_to_add, add_same_located_points, nb_in_seq = nil)`

First parameter is the route where we want to insert the point,
Second contains the computed data regarding this point insertion,
Third is a boolean which says if we need to insert point at same location when inserting this point,
Fourth is the number of the visit. 

There may be a naming problem : best_index is not an integer even though it contains the best_index value among other things.